padertorch/contrib/examples/source_separation/tasnet/train.py

```diff
@@ -142,6 +134,29 @@ def stft():
     }
 
 
+@ex.named_config
+def dprnn():
+    trainer = {'model': {'separator': {
+        'factory': pt.modules.dual_path_rnn.DPRNN,
+        'input_size': 64,
+        'rnn_size': 128,
+        'window_length': 100,
+        'hop_size': 50,
+        'num_blocks': 6,
+    }}}
+
+
+@ex.named_config
+def convnet():
+    feat_size = 256
+    trainer = {'model': {'separator': dict(
```

Comment:
Why is this `dict`, but for the dprnn it is `{}`?

Reply:
it's due to copy and paste, but I change it to be more similar